backend/tet/requirements.in

```diff
@@ -9,7 +9,7 @@ django-simple-history
 django-storages[azure]
 djangorestframework
 django~=3.2
-drf-oidc-auth
+drf-oidc-auth==0.10.0  # For GDPR API to work, otherwise got 'Invalid claim: "amr"'
```

Comment:
This is almost 3 years old and not even a major version. I do understand the issue there, but a authentication is such a critical area where I would follow the latest major versions when there are those available.

In Kultus and Kukkuu there is an alternative fix for this, which converts the amr to list when it is not:
https://github.com/City-of-Helsinki/palvelutarjotin/blob/master/palvelutarjotin/oidc.py#L4. It should be easy and simple to activate in this project too.

Reply:
Ok, sounds good to fix before production release. I will change this before releasing to production.